src/debug/ee/functioninfo.cpp

```diff
@@ -2253,7 +2282,7 @@ void DebuggerMethodInfoTable::ClearMethodsOfModule(Module *pModule)
             DebuggerMethodInfo * dmi = entry->mi;
             while (dmi != NULL)
             {
-                DebuggerJitInfo * dji = dmi->GetLatestJitInfo_NoCreate();
```

Comment:
If GetLatestJitInfo automatically has NoCreate semantics then this seems fine, but if GetLatestJitInfo continues to automatically populate the cache (as I'll suggest elsewhere) then this seems like a potential performance issue creating datastructures in order to destroy them.

Reply:
I'll revert this based on other feedback